python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -826,6 +829,36 @@ def do_evaluate(
                     plc.interop.from_arrow(pa.scalar(stop, type=pa.int32())),
                 )
             )
+        elif self.name in {
+            pl_expr.StringFunction.StripChars,
+            pl_expr.StringFunction.StripCharsStart,
+            pl_expr.StringFunction.StripCharsEnd,
+        }:
+            column = self.children[0].evaluate(df, context=context, mapping=mapping)
+            expr_chars = self.children[1]
+            assert isinstance(expr_chars, Literal)
+
+            if expr_chars.value == pa.scalar(""):
+                # No-op in python, polars, but libcudf removes whitespace so return early
+                return column
+            elif expr_chars.value == pa.scalar(None):
+                # Polars api for chars:
+                # If set to None (default), all leading and trailing whitespace is removed
+                # libcudf api for chars:
+                # If to_strip is the empty string, whitespace characters are removed
+                chars = plc.interop.from_arrow(pa.scalar("", type=pa.string()))
```

Comment:
Does the set of whitespace that polars strips by default match what cudf does?

Can we propose turning `None` into the default set in the IR preprocessing in polars?

Reply:
The docs for libcudf state 
```
Whitespace is considered the space character plus control characters like tab and line feed.
```
I think our tests cover these cases. I am not sure what you mean by `Can we propose turning None into the default set in the IR preprocessing in polars?`. Do you just mean as above, that we'd rather adjust the scalar during the planning phase rather than the execution phase?